### ISSUE TITLE
No way to tell a post is a reply in Ivory

### ISSUE THREAD

**Issue by jvns** [videah/SkyBridge#12]
In Mastodon, the way you generally tell that a post is a reply is that it has @someuser in the text of the post.

In Bluesky, this isn’t the case — bsky.app instead has a separate “replying to” UI element that shows that the post is a reply

It doesn’t seem obvious how to reconcile these (if the bridge puts  @someuser in the text of the post, replying to that reply in Ivory wouldn’t work well because then you’ll end up with an extra @someuser in the reply)

**Comment by videah** [comment]
Yeah this has been something I've been pondering the past couple of days, I'm not sure what the best way to go around emulating this behaviour is without causing the other issues you mentioned. Will have a deeper think about it soon because it can be quite confusing, especially on timelines.

**Comment by mergesort** [comment]
I don't know if you've thought about this more but assuming Bluesky provides something similar to Twitter's `reply_to` field in the API, you could prepend the username to the Bluesky post like so:
> Hello world

Becomes:
> @reply_to_username Hello world

Does that sound like a reasonable improvement?
